source/core/Console.cs

```diff
@@ -34,107 +34,43 @@ namespace GTA
 	//Used to describe console args (Needed for creating a Help-func)
 	internal class ConsoleArg
 	{
-		#region Fields
-		string _type;
-		string _name;
-		#endregion
-
 		public ConsoleArg(string type, string name)
```

Comment:
We could get rid of this constructor altogether and make the auto properties below settable.

Reply:
If we do that, we have to write like this, don't we?
```
	var arg = new ConsoleArg();
	arg.Type = args.ParameterType.Name;
	arg.Name = args.ParameterType.Name;

	command.ConsoleArgs.Add(arg);
```

Reply:
Ignore my previous comment. I mixed up the class with the console command attribute. Doesn't make sense to have setters on this one actually.

Reply:
Okay, reverted the commit.